src/main/kotlin/no/nav/k9/tjenester/avdelingsleder/AvdelingslederTjeneste.kt

```diff
@@ -305,7 +305,8 @@ class AvdelingslederTjeneste(
 
                 val oppgave = oppgaveRepository.hent(uuid)
 
-                if (configuration.koinProfile() != KoinProfile.LOCAL && !pepClient.harTilgangTilLesSak(
+                if (configuration.koinProfile() != KoinProfile.LOCAL &&
+                    !oppgave.harFagSaksNummer() || !pepClient.harTilgangTilLesSak(
                         fagsakNummer = oppgave.fagsakSaksnummer,
                         aktørid = oppgave.aktorId
                     )
```

Comment:
Her trenger du parantes rundt uttrykket for å unngå tilgangskontrollsjekk lokalt (fordi || har lavere presendens enn &&):
```
(!oppgave.harFagSaksNummer() || !pepClient.harTilgangTilLesSak(
    fagsakNummer = oppgave.fagsakSaksnummer,
    aktørid = oppgave.aktorId
))
```
...eller bedre: Trekk ut i egen metode.

Reply:
Ah, det stemmer ja. Lager en metode for det både her og i OppgaveTjeneste